src/components/Footer/Footer.tsx

```diff
@@ -1,5 +1,45 @@
-import styles from "./footer.module.css"
+import styles from "./footer.module.css";
+import {
+  FaFacebookSquare,
+  FaInstagramSquare,
+  FaWhatsappSquare,
+} from "react-icons/fa";
+import { Link } from "react-router-dom";
 
 export default function Footer() {
-    return <div id="footer" className={styles.mainFooterContainer}></div>
-}
\ No newline at end of file
+  return (
+    <div id="footer" className={styles.mainFooterContainer}>
+      <div id="socials" className={styles.socialsContainer}>
+        <div id="facebook" className={styles.indivSocial}>
+          <FaFacebookSquare size="2rem" color="black"></FaFacebookSquare>
+          <Link to="/">Facebook</Link>
+        </div>
+        <div id="instagram" className={styles.indivSocial}>
+          <FaInstagramSquare size="2rem" color="black"></FaInstagramSquare>
+          <Link to="/">Instagram</Link>
+        </div>
+        <div id="whatsapp" className={styles.indivSocial}>
+          <FaWhatsappSquare size="2rem" color="black"></FaWhatsappSquare>
+          <Link to="/">WhatsApp</Link>
+        </div>
+      </div>
+      <div id="pagelist" className={styles.pageListContainer}>
+        <Link to="/about">Sobre</Link>
+        <Link to="/about">Bebé</Link>
+        <Link to="/about">Flores</Link>
+        <Link to="/about">Acessórios</Link>
+        <Link to="/about">Anjinhos</Link>
+        <Link to="/about">Contactos</Link>
```

Comment:
Be careful, links are all to the same location! I prefer that, for now, the Link tag to not have the "to" argument
Just leave it empty

Reply:
Done, I removed it all, check in here [3118a5f](https://github.com/ozefreitas/flordelinha_website/pull/15/commits/3118a5f2b6a49b31b84adaa3420c8a96ec05b220)